### PR TITLE
Use SbThreadSleep for cval_stats_test.

### PR DESCRIPTION
**Pull request by sideb0ard**
This should fix test on Evergreen.

b/292030213

Change-Id: I4514aa08729fa814f58a3f2cf1b7b0339e59574d

### CODE REVIEW

File: cobalt/media/base/cval_stats_test.cc [hunk 1]
```diff
@@ -71,7 +57,8 @@ TEST(MediaCValStatsTest, NothingRecorded) {
   CValStats cval_stats_;
 
   cval_stats_.StartTimer(MediaTiming::SbPlayerCreate, kPipelineIdentifier);
-  sleep_ms(kSleepTimeMs);
+  SbThreadSleep(kSleepTimeMs);
```
Comment (oxve) [inline cobalt/media/base/cval_stats_test.cc:60]: SbThreadSleep expects SbTime. I think the constant should reflect that.

Kinda funny that the tests still passed with the sleep reduced by 99.9%

Comment (sideb0ard) [reply cobalt/media/base/cval_stats_test.cc]: updated with SbTime.
Tests just confirm that if time has passed, a change is recorded, so thats understandable about them still passing with a different time.

File: cobalt/media/base/cval_stats_test.cc [hunk 2]
```diff
@@ -33,21 +29,11 @@ const char kCValnameMinimum[] = "Media.SbPlayerCreateTime.Minimum";
 
 const char kPipelineIdentifier[] = "test_pipeline";
 
-constexpr int kSleepTimeMs = 50;
+constexpr SbTime kSleepTimeSbTime = 50;  // 50 microseconds
```
Comment (xiaomings) [inline cobalt/media/base/cval_stats_test.cc:32]: nit: kSleepTimeSbTime => kSleepTime, as `SbThreadSleep(kSleepTime)` is descriptive enough.

**Review by xiaomings — APPROVED** [review]

File: cobalt/media/base/cval_stats_test.cc (hunk 2 above)
Comment (sideb0ard) [reply cobalt/media/base/cval_stats_test.cc]: updated

**Review by xiaomings — APPROVED** [review]

**Review by oxve — APPROVED** [review]

**Pull request merged**